spring-cloud-dataflow-build/spring-cloud-dataflow-build-dependencies/pom.xml

```diff
@@ -21,6 +21,7 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<!-- Keep spring boot version in sync between spring-cloud-dataflow-build and spring-boot-dependencies (parent and properties) -->
 		<spring-boot.version>2.7.18</spring-boot.version>
+		<spring-framework.version>5.3.32</spring-framework.version>
```

Comment:
Please split the SF and P updates into separate commits.

Reply:
I see you have the commit message specifying this but it would be simple enough to just split into 2 commits and keep in this single PR is fine.